corehq/apps/custom_data_fields/edit_model.py

```diff
@@ -59,7 +59,7 @@ class XmlSlugField(forms.SlugField):
     default_validators = [
         validate_slug,
         validate_reserved_words,
-        RegexValidator(r'^[a-zA-Z]', ''),
+        RegexValidator(r'\D', ''),
```

Comment:
This is changing from "the string must begin with a letter" to "there is a non-digit in this string" is that correct? may be useful to add a comment/tests describing what this intended to catch.

If my understanding is correct then I approve.

Reply:
yep, that's it. added comment.